### PR TITLE
feat(sdk-core): add support for ETH TSS staking

### PR DESCRIPTION
**Pull request by noel-bitgo**
BOS-627

### CODE REVIEW

File: modules/sdk-core/src/bitgo/wallet/wallet.ts [hunk 1]
```diff
@@ -115,6 +115,12 @@ export class Wallet implements IWallet {
     }
   }
 
+  isEthTss(): boolean {
```
Comment (mmcshinsky-bitgo) [inline modules/sdk-core/src/bitgo/wallet/wallet.ts:118]: Can we do the same as https://github.com/BitGo/BitGoJS/blob/master/modules/sdk-core/src/bitgo/wallet/wallets.ts#L182 until we see the use case expand in being used in multiple places?

**Review by mmcshinsky-bitgo — CHANGES_REQUESTED** [review]

File: modules/sdk-core/src/bitgo/wallet/wallet.ts (hunk 1 above)
Comment (noel-bitgo) [reply modules/sdk-core/src/bitgo/wallet/wallet.ts:118]: I am not sure I understand what you're suggesting? 🤔

Comment (mmcshinsky-bitgo) [reply modules/sdk-core/src/bitgo/wallet/wallet.ts:118]: Can we move this logic into a variable without adding a new method in the `build` method the same way we are doing it in `generateWallet`?

Comment (noel-bitgo) [reply modules/sdk-core/src/bitgo/wallet/wallet.ts]: just pushed a change, let me know if this is what you meant

File: modules/sdk-core/src/bitgo/wallet/wallet.ts [hunk 2]
```diff
@@ -2107,7 +2107,11 @@ export class Wallet implements IWallet {
     if (!this.baseCoin.supportsStaking()) {
       throw new Error(`Staking not supported for ${this.coin()}`);
     }
-    return new StakingWallet(this);
+    const isEthTss =
+      this.baseCoin.getFamily() == 'eth' && this._wallet.coinSpecific?.walletVersion
```
Comment (mmcshinsky-bitgo) [inline modules/sdk-core/src/bitgo/wallet/wallet.ts:2111]: Does this need a to check for `teth` and `geth` or are they all returning `eth`?

Comment (noel-bitgo) [reply modules/sdk-core/src/bitgo/wallet/wallet.ts:2111]: as far as I can tell the coin family is eth

**Review by mmcshinsky-bitgo — APPROVED** [review]

**Pull request merged**